:sparkles: feature/add-tags

nuuvia opened the pull request.

nuuvia committed:
:sparkles: feature/add-tags

nuuvia committed:
:sparkles: feature/add-tags

IrynaShostenko committed:
feature/create-starter-table

- implemented table with allowed commands and examples

nuuvia committed:
:sparkles: enhancement/divide-adding-tags-into-two-steps

First, the user writes a title for the note. Then, they add tags in a separate step.

nuuvia committed:
:sparkles: enhancement/divide-adding-tags-into-two-steps

First, the user writes a title for the note. Then, they add tags in a separate step.

nikita-parkhomenko committed:
feature/Save-and-load-books-data

- implement serialize/deserialized function for address book and note book
- improvements

IrynaShostenko committed:
feature/create-starter-table

- improvements

IrynaShostenko committed:
Merge pull request #20 from nikita-parkhomenko/feature/create-starter-table

feature/create-starter-table

nikita-parkhomenko committed:
feature/Save-and-load-books-data

- add files to gitignore

nikita-parkhomenko committed:
feature/Save-and-load-books-data

- resolve conflict

nikita-parkhomenko committed:
Merge pull request #21 from nikita-parkhomenko/feature/Save-and-load-books-data

feature/Save-and-load-books-data

nikita-parkhomenko committed:
enhancement/Improvements

- improvements

nikita-parkhomenko committed:
enhancement/Improvements

- improvements

nikita-parkhomenko committed:
Merge pull request #22 from nikita-parkhomenko/enhancement/Improvements

enhancement/Improvements

nikita-parkhomenko committed:
enhancement/Edit-note-improvements

- edit note fix

nikita-parkhomenko committed:
enhancement/Edit-note-improvements

- add spacing for input

nikita-parkhomenko commented on main.py:
```diff
@@ -482,6 +512,9 @@ def main():
         elif command == "delete-note":
             delete_note(args, note_book)
 
+        elif command == "add-tag":
```
```suggestion
        elif command == "note-add-tag":
```

andrii248 committed:
feature/Search-note

- implemented search note feature
- added the feature to the List of bot commands

nikita-parkhomenko committed:
Merge pull request #23 from nikita-parkhomenko/enhancement/Edit-note-improvements

enhancement/Edit-note-improvements

nikita-parkhomenko commented on main.py:
```diff
@@ -173,6 +180,13 @@ def delete_note(self, title):
             return f"[cornflower_blue]Note[/cornflower_blue] '{title}' [cornflower_blue]has been deleted.[/cornflower_blue]"
         return f"[indian_red]Note[/indian_red] '{title}' [indian_red]was not found.[/indian_red]"
 
+    def add_tag_to_note(self, title, tag):
```
this method is useless and is not called anywhere, do you really need it?

nikita-parkhomenko committed:
Update main.py

nikita-parkhomenko committed:
Merge pull request #24 from nikita-parkhomenko/feature/Search-note

feature/Search-note

nuuvia committed:
test
